fv3core/utils/corners.py

```diff
@@ -1,96 +1,154 @@
 from gt4py.gtscript import PARALLEL, computation, horizontal, interval, region
 
+import fv3core._config as spec
 import fv3core.utils.gt4py_utils as utils
 from fv3core.decorators import gtstencil
+from fv3core.utils.typing import FloatField
 
 
 sd = utils.sd
 
 
-def fill_4corners_x(q: sd):
+def fill_corners_cells_mult_x(
```

Comment:
If "cells" means the center of the grid cell, it might be clearer to say "center". This can later contrast with e.g. "corner" or "y_interface".

It's a shame though that these things can't be parametrized.

Reply:
I think I found a way that we can parameterize it. Let's chat about it next week.